### PR TITLE
Add examples for std::ascii

### PR DESCRIPTION
**Pull request by steveklabnik**
Also tweaked a few things.

### CODE REVIEW

File: src/libstd/ascii.rs [hunk 1]
```diff
@@ -246,7 +324,7 @@ pub struct EscapeDefault {
     data: [u8; 4],
 }
 
-/// Returns a 'default' ASCII and C++11-like literal escape of a `u8`
+/// Returns an iterator that produces an escaped version of a `u8`.
```
Comment (Manishearth) [inline src/libstd/ascii.rs]: 'default' is mentioned later too, could it be given some better context/antecedent?

**Comment by Manishearth** [comment]
r=me with or without that nit fixed :)

File: src/libstd/ascii.rs (hunk 1 above)
Comment (steveklabnik) [inline src/libstd/ascii.rs]: I'm not sure what it means, actually. But you're right it needs one...

**Comment by steveklabnik** [comment]
Ehhh let's just leave the nit for now

**Pull request merged**